package-lock.json

```diff
@@ -1688,13 +1688,13 @@
     "buffer-crc32": {
       "version": "0.2.13",
       "resolved": "https://registry.npmjs.org/buffer-crc32/-/buffer-crc32-0.2.13.tgz",
-      "integrity": "sha512-VO9Ht/+p3SN7SKWqcrgEzjGbRSJYTx+Q1pTQC0wrWqHx0vpJraQ6GtHx8tvcg1rlK1byhU5gccxgOgj7B0TDkQ==",
+      "integrity": "sha1-DTM+PwDqxQqhRUq9MO+MKl2ackI=",
```

Comment:
Not that I think it matters much in practice, but: why, in the year of our Linux on the Desktop 2022 are these going back to sha1?

Maybe try nuking your npm cache via the steps at https://dnasir.com/2018/02/12/solving-the-package-lock-json-integrity-discrepancy/ ?